### PR TITLE
Let station AI use long range fax machines

### PR DESCRIPTION
**Pull request by NoElkaTheGod**
## About the PR
Station AI can now interact with long range fax machines. You can't insert paper into the tray (because you have no hands), but at least you can print something (yes, the "print from file" button actualy works).
It would be cool to give the AI an ingame text editor when interacting with fax machines, but i won't do that now because _reasons_.

## Why / Balance
1. Paperwork is good, but paperwork done by AI is better.
2. Could create some interesting RP situations.
3. Why not?

## Technical details
Added StationAIWhitelist component to the FaxMachineBase entity.

## Media
изображение

## Requirements
- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.

**Changelog**
:cl:
- tweak: Station AI can now interact with long range fax machines.

### CODE REVIEW

**Review by slarticodefast — APPROVED** [review]
Good idea, gives the AI a method to communicate with the crew in secret.
Tested in-game, works as intended.

File: Resources/Prototypes/Entities/Structures/Machines/fax_machine.yml [hunk 1]
```diff
@@ -4,6 +4,7 @@
   name: long range fax machine
   description: Bluespace technologies on the application of bureaucracy.
   components:
+  - type: StationAiWhitelist
```
Comment (slarticodefast) [inline Resources/Prototypes/Entities/Structures/Machines/fax_machine.yml:7]: Oh, one thing to clean up:
Move it a little lower in the list, components are usually ordered by importance and grouped thematically.
So the Sprite should stay on top. I would suggest move `StationAiWhitelist` below `UserInterface`.

Comment (NoElkaTheGod) [reply Resources/Prototypes/Entities/Structures/Machines/fax_machine.yml]: Didn't know that, thanks. Changes applied.

**Review by slarticodefast — APPROVED** [review]
Perfect! Thank you for your first contribution :)

**Pull request merged**